.github/workflows/uploadFirrtlReleaseArtifacts.yml

```
@@ -1,11 +1,38 @@
-name: Upload Firrtl Binaries
+name: Upload Firrtl Release Artifacts
 
 on:
   release:
     types: [created]
   workflow_dispatch:
 
 jobs:
+  publish-sources: 
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-20.04
+    steps:
+      # Clone the CIRCT repo and its submodules. Do shallow clone to save clone
+      # time.
+      - name: Get CIRCT and LLVM
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+          submodules: "true"
+
+      # Package up sources for distribution, as the default source bundles from GitHub don't include LLVM. Only runs on the ubuntu-20.04 runner.
```

```suggestion
      # Package up sources for distribution, as the default source bundles from GitHub don't include LLVM.
```
I think that comment is no longer relevant